modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -987,6 +991,61 @@ private void _testPatchAccountWithContactInformation() throws Exception {
 			patchAccount.getAccountContactInformation());
 	}
 
+	private void _testPatchAccountWithEmptyOrganizationExternalReferenceCodeArray()
+		throws Exception {
+
+		Account randomAccount = randomAccount();
+
+		Organization organization1 = OrganizationTestUtil.addOrganization();
+
+		Organization organization2 = OrganizationTestUtil.addOrganization();
+
+		randomAccount.setOrganizationExternalReferenceCodes(
+			new String[] {
+				organization1.getExternalReferenceCode(),
+				organization2.getExternalReferenceCode()
+			});
+
+		Account postAccount = _postAccount(randomAccount);
+
```

Comment:
can we add an assert to check that now we have tow organizations in the postAccount?
I want also another assert because
* in patch if we don't pass the arrays at all it should keep the previous values
* in put if we don't pass the arrays at all it should reset them (like passing empty arrays)
---------------------
so in the patch tests we should do
add two
assert 2
patch without arrays
assert 2
patch with empty array
assert 0

---------------
so in the put tests we should do
add two
assert 2
patch without arrays
assert 0
put with 2
assert 2
patch with empty array
assert 0

Reply:
@smottal but for put to work differently than patch, I have to completely change the ResourceImpl logic.

Reply:
I can't use the same getOrgIds method

Reply:
for now is fine ... put the scenario only for patch method